README.md

```diff
@@ -2,7 +2,7 @@
 
 # TechnicalIndicators
 
-A javascript technical indicators written in javascript. 
+A javascript technical indicators written in javascript.
```

Comment:
Sorry. My editor trims these random spaces. Since there is no value in random extra spaces I have let the changes in